dbfit-java/sqlserver/src/test/resources/acceptancescripts-SqlServer.sql

```diff
@@ -208,4 +247,11 @@ as
 begin
 set @copyOfInParam = @inParam
 set @constOutParam = 123.456;
-end
\ No newline at end of file
+end
+GO
+
+create procedure [dbo].[MakeUser] AS
+begin
+	insert into Users (Name,UserName) values ('user1','fromproc');

```

Comment:
Missing white-space between `Name,` and `Username`; and also between the values.